flink-ml-core/src/test/java/org/apache/flink/ml/linalg/VectorsTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.linalg;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+/** Tests the behavior of Vectors. */
+public class VectorsTest {
```

Comment:
If this test is supposed to cover only `SparseVector`, could we name it as `SparseVectorTest`? Otherwise, could we specify in the test method name whether the test is for `SparseVector` or `DenseVector`?

Reply:
I prefer renaming `VectorsTest` to `SparseVectorTest`, since adding `SparseVector` to the name of all test cases seems redundant. I'll change the code in this way.